<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Strobe_Settings_Screen</class>
 <widget class="QWidget" name="HNS_Strobe_Settings_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_StrobeSet_Save">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>600</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Cancel">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>600</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_StrobeSet_Pattern">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLineEdit:disabled{
	background-color:#FFFFFF;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Enable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>ENABLED</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_StrobeSet_Speed">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLineEdit:disabled{
	background-color:#FFFFFF;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Duration_Up">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Test">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Pattern_Dn">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>380</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>DOWN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_82">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>200</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PATTERN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_StrobeSet_Duration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLineEdit:disabled{
	background-color:#FFFFFF;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Duration_Dn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>DOWN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_SpeedMode">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>610</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Offset
VSL</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_80">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SPEED</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Pattern_Up">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Speed_Up">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_StrobeSet_Speed_Dn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>DOWN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_81">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DURATION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
